src/main/java/carpet/script/language/Operators.java

```diff
@@ -29,12 +29,12 @@
         put("multiplication*/%", 30);
         put("addition+-", 20);
         put("compare>=><=<", 10);
-        put("equal==!=", 7);
-        put("and&&", 5);
-        put("or||", 4);
-        put("assign=<>", 3);
-        put("def->", 2);
-        put("nextop;", 1);
+        put("equal==!=", 8);
+        put("and&&", 4);
+        put("or||", 3);
+        put("assign=<>", 2);
+        put("def->", 1);
+        put("nextop;", 0);
```

Comment:
If the operators aren't included in the end, shouldn't this be reverted?

Reply:
oh yeah...